tools/src/h5import/h5import.c

```diff
@@ -2536,7 +2536,7 @@ parsePathInfo(struct path_info *path, char *temp)
 
     token = HDstrtok(temp, delimiter);
     if (HDstrlen(token) >= MAX_PATH_NAME_LENGTH) {
-        (void)HDfprintf(stderr, err1);
+        (void)HDfprintf(stderr, "%s", err1);
```

Comment:
The problem here is that there is missing data, the err1 is:
    const char *err1  = "Invalid number of arguments:  %d.\n";

Reply:
No the problem is that a global was redefined!
    const char *err1 = "Path string larger than MAX_PATH_NAME_LENGTH.\n";

Mayber rename the new err1 to err_max to not conflict.